test/ex_binance/spot/private/trades_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule ExBinance.Spot.Private.TradesTest do
+  use ExUnit.Case
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+
+  alias ExBinance.Spot.Private.Trades
+
+  setup_all do
+    HTTPoison.start()
+  end
+
+  @credentials %ExBinance.Credentials{
+    api_key: System.get_env("BINANCE_API_KEY"),
+    secret_key: System.get_env("BINANCE_API_SECRET")
+  }
+
+  describe "historic_trades" do
+    test "can query historic trades" do
+      use_cassette "spot/private/historic_trades" do
+        assert {:ok, trades} = Trades.historic_trades("ETHBTC", 0, 10, @credentials)
+
+        assert Enum.count(trades) == 10
+
+        [
+          %ExBinance.Trade{
```

Comment:
I feel like we don't need to assert on the value for each of these fields. How do you feel about changing the assertion so that it just checks that a property like `quote_qty != nil`

Reply:
Yeah sure, updated the code and asserted that the field exist with the correct type of values

Thank you for the quick turnaround for a review